### ISSUE TITLE
Confuse of motivation

### ISSUE THREAD

**Issue by superhero-7** [FreddeFrallan/Multilingual-CLIP#36]
Thank you for your nice work. But I still have a question: the training data is entirely rely on tranlation, then what is the differen between mulitilingual CLIP and pure English CLIP + a translation model ? We can completely use a translation model plus English CLIP to realize the function of multi-language CLIP. I hope you can give me some inspiration, thank you in advance.

**Comment by Lauler** [comment]
It is true. You can use a translation model to translate to English. 

I would argue that in a real world use scenario it may still be beneficial to use model distillation for a couple of reasons:

1. Inference speed and latency in production use. Inference is much faster when a user's input in the form of a search query can go straight to the CLIP model, as opposed to first being piped through a translation model before it can be sent to CLIP.
2. Most user queries will be very short, often single words. Translation models' performance suffer without proper context, when individual words have multiple definitions that are valid in different contexts. Training a distilled model on translated sentences alleviates some of that ambiguity for the translation model.

Though, you do have a point that using a translation model often can suffice!

**Comment by superhero-7** [comment]
Yeah. Thanks for replying so fast. I totally argree the inference speed. But why does the distilled model perform better when the query is a single word? In my understanding, if the training data is all translated, the model's capabilities should be completely equivalent to translation plus English CLIP. 

By the way, does the distillation model have advantages in other scenarios?

**Comment by rom1504** [comment]
If you do the translation+ English clip inference on a single sample then
you can use the knowledge of the translation model, and the similarity
knowledge of the English clip model. There is no similarity knowledge of
(image, non English) known.
It may make a difference to use the knowledge of (image, non English)
gained during training on more non English examples.

That said, did you check out the multilingual models at openclip ? I
believe they are stronger than the ones in this repo

**Comment by superhero-7** [comment]
I see. I can understand that some additional knowledge will be provided when there are some pure multi-lingual data in the training corpus, but if it is all translation, it seems to be no different from splicing a translation model. I am also engaged in multi-language related work, but I have always been confused about the meaning of this matter, so I want to discuss it with you. Do we really need multilingual CLIP? Or can all multilingual work be replaced by translation?

**Comment by rom1504** [comment]
If we had a perfect translation model that would go from any language to English while keeping all the nuance and the culture relevance as well as what it may mean visually, then indeed multilingual clip would not be useful.

In practice I think we don't have that. So I think learning multilingual clip is quite useful. We have seen at clip benchmark repo for example that results of most multilingual clip on translated imagenet classes are quite poor which seem to mean a lot of progress could be made there.

**Comment by superhero-7** [comment]
Thanks a lot! I think it makes sense.
